api/src/main/java/org/pmiops/workbench/firecloud/FireCloudConfig.java

```diff
@@ -41,16 +43,30 @@ public ApiClient fireCloudApiClient(UserAuthentication userAuthentication,
     return apiClient;
   }
 
+  private String getAccessToken(WorkbenchEnvironment workbenchEnvironment) throws IOException {
+    // When running locally, we get application default credentials in a different way than
+    // when running in Cloud.
+    if (workbenchEnvironment.isDevelopment()) {
+      GoogleCredential credential = GoogleCredential.getApplicationDefault()
```

Comment:
This builds in an assumption that all developers directly have access to the free tier billing account. Is that safe to assume / currently true? (technically they currently have it via access to SA creds in GCS)

Reply:
Not quite. When you run dev-up, GOOGLE_APPLICATION_CREDENTIALS is set to sa-key.json in the api dir, which is the service account credentials for test. That will have access to the free tier billing account for nonprod. So developers don't need direct access.